app/src/main/kotlin/com/wire/android/ui/authentication/devices/remove/RemoveDeviceViewModel.kt

```diff
@@ -84,12 +84,14 @@ class RemoveDeviceViewModel @Inject constructor(
         viewModelScope.launch {
             state = state.copy(isLoadingClientsList = true)
             val selfClientsResult = fetchSelfClientsFromRemote()
-            if (selfClientsResult is SelfClientsResult.Success) {
-                state = state.copy(
+            state = if (selfClientsResult is SelfClientsResult.Success) {
+                state.copy(
                     isLoadingClientsList = false,
                     deviceList = selfClientsResult.clients.filter { it.type == ClientType.Permanent }.map { Device(it) },
                     removeDeviceDialogState = RemoveDeviceDialogState.Hidden
                 )
+            } else {
+                state.copy(isLoadingClientsList = false, error = RemoveDeviceError.InitError)
```

Comment:
In this case, I guess it is fine to display the error message since this is used directly after login and there is no local data, we just need to make sure the dialog is dismissible and the user can close it to take an action like logout

Reply:
Dialog is dismissible, there are two buttons: "retry" and "cancel" which executes the same thing as if the user pressed system back button or X on top bar - shows another dialog asking whether the user wants to cancel the action and logout.